lib/compiler/hooks/tsconfig-paths.hook.ts

```diff
@@ -55,7 +55,6 @@ export function tsconfigPathsBeforeHookFactory(
                   )
                 : tsBinary.factory.updateImportDeclaration(
                     node,
-                    node.decorators,
```

Comment:
Will this work for older versions too?

Reply:
It should work for versions at least since 4.8 since that's when this was deprecated https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-8.html#decorators-are-placed-on-modifiers-on-typescripts-syntax-trees

Reply:
Is the new diff more satisfactory?